README.md

```diff
@@ -72,19 +78,47 @@ make build_tests_standalone -j
 ctest --timeout 120 # currently some tests might hang
 ```
 
-## Testing
+## Environment Flags
+
+### CHIP_BE
+
+Select the backend to execute on.
+Possible values: opencl, level0(default)
```

Comment:
Currently CHIP-SPV defaults to OpenCL - at least on systems with both the OpenCL and Level Zero available.